components/eam/cime_config/config_component.xml

```diff
@@ -152,10 +151,10 @@
       <!-- MMF / Super-Parameterization -->
       <value compset="2000(?:SOI)?_EAM%MMF[1XO]">2000_cam5_av1c-04p2-MMF-1mom</value>
```

Comment:
I thought we changed these use cases to be consistent with the switch to F2010, but I guess maybe that never made it in. We should fix this in a subsequent PR.

Reply:
Yea, I noticed this the other day, the white space changes below this caused a conflict when I tried to merge it with master. Just make sure to keep the new use case file names and we shoudl be fine.